src/main/java/org/commcare/modern/database/DatabaseHelper.java

```diff
@@ -34,6 +34,20 @@
     public static Pair<String, String[]> createWhere(String[] fieldNames, Object[] values,
                                                      EncryptedModel em, Persistable p)
             throws IllegalArgumentException {
+        return createWhere(fieldNames, values, null, null, em, p);
```

Comment:
As per the below comment in `createWhere`, my intuition is that these should be instantiated as empty arrays, not null values.